composer.json

```diff
@@ -17,10 +17,10 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=5.5"
+        "php": ">=7.1"
     },
     "require-dev": {
-        "phpunit/phpunit": ">=4.1 <8",
+        "phpunit/phpunit": ">=7 <9",
```

Comment:
How about removing the upper bound ?

Reply:
We have a lot of `assertInternalType` assertions that were removed from PHPUnit 9.

Maybe I can add some static assertion constructor that, depending on the PHPUnit version, call the right assertion. Let me try.

Reply:
Maybe something like this https://github.com/giorgiosironi/eris/pull/148/commits/9a4990f8014423b5ca9ff7074ff3c544318e23d2

Reply:
The idea is to remove the `<9`, see: https://github.com/symfony/symfony/pull/36876

Reply:
removed